addon/index.js

```diff
@@ -157,6 +158,15 @@ function initServerEnvironmentPreference() {
   });
 }
 
+function openOnboardingTab() {
+  // TODO: Settings is populated async, so we need a global afterSettingsReady
+  // promise that this function can wait on.
+  tabs.open({
+    url: SERVER_ENVIRONMENTS.production.BASE_URL + '/onboarding',
```

Comment:
@lmorchard I was actually thinking of passing a callback to updatePrefs to call when it was totally done, simpler than a promise and does what's needed. THoughts?